gammapy/cube/tests/test_simulate.py

```diff
@@ -116,3 +117,19 @@ def test_MDE_sample_background():
     assert_allclose(bkg_evt.table["RA"][0], 265.7253792887848, rtol=1e-5)
     assert_allclose(bkg_evt.table["DEC"][0], -27.727581635186304, rtol=1e-5)
     assert_allclose(bkg_evt.table["MC_ID"][0], 0, rtol=1e-5)
+
+
+@requires_data()
+def test_MDE_sample_edisp():
+    edisp_map = make_edisp_map_test()
+
+    dataset = dataset_maker()
+    sampler = MapDatasetEventSampler(random_state=0)
+    events = sampler.sample_sources(dataset=dataset)
+    events = sampler.sample_edisp(edisp_map, events)
+
+    assert len(events.table) == 726
+    assert_allclose(events.table["ENERGY_TRUE"][0], 1.0600765557667795, rtol=1e-5)
```

Comment:
Can you add an additional assert on the unit of the columns, like I did here: https://github.com/gammapy/gammapy/pull/2659/commits/7c94768ac05b548c818ed4af17cfe46f72a1ed75

Reply:
Done.